Woof/Woof/ViewModel/DetailPetSitterViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+/// The view model for a detailed sitter card, responsible for preparing and providing data for the view.
+final class DetailSitterViewModel: ObservableObject {
+    /// The full name of the sitter.
```

Comment:
You have just duplicated the model that already exists. 
I think we should discuss whether can we reuse `SitterCardViewModel` or should create a separate model for that view.
Also, I think we should format the phone number to display it the view in specific format (for example with + sign at the beginning)

Reply:
As we discussed, we need two separated viewModels.